sympy/physics/control/lti.py

```diff
@@ -4845,6 +4845,161 @@ def append(self, other):
         D[self.num_outputs:, self.num_inputs:] = other._D
         return StateSpace(A, B, C, D)
 
+    def _calc_orthogonal_complement(self, M, dim):
+        """
+        Returns a basis of the orthogonal complement of a subspace
+        represented by the matrix M.
+        The orthogonal complement is computed as the null space of the
+        transpose of M.
+
+        """
+        if M.shape[0] == 0:
+            return eye(dim).columnspace()
+
+        return M.T.nullspace()
+
+    def apply_similarity_transform(self, transform_matrix):
+        r"""
+        Returns an algebrically equivalent state space model, based on the
+        transformation matrix `T` such that:
+
+        .. math::
+            \begin{cases}
+            \bar A=T^{-1}AT\\
+            \bar B=T^{-1}B\\
+            \bar C= CT\\
+            \bar D=D\end{cases}
+
```

Comment:
Please follow our docstring styleguide: https://docs.sympy.org/latest/contributing/docstring.html This needs a "Parameters" section and "Returns" section, for example.